requirements.txt

```diff
@@ -2,7 +2,7 @@
 certifi
 cffi
 gunicorn
-pyramid_multiauth
+honcho
```

Comment:
This looks like a rebase error. There should be no need for `honcho` in `requirements.txt` any more.